domains/Discussions/Controllers/QuestionsGetAnswersController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Question;
+use  Illuminate\Database\Eloquent\Builder;
+use Illuminate\Contracts\Auth\Factory as Auth;
+use Illuminate\Contracts\Pagination\Paginator;
+
+class QuestionsGetAnswersController extends Controller
+{
+private Question $question;
+
+    public function __construct(Auth $auth, Question $question)
+    {
+        if ($auth->guest()) {
+            $this->middleware('throttle:30,1');
+        }
+
+        $this->question = $question;
+    }
+
+    public function __invoke(int $id, Request $request): Paginator
```

Comment:
We need to return a question resource like other controllers are doing.

Reply:
Nit: switch the order of the function parameters. It works both ways, it is just to keep it aligned with the other controllers.